app/assets/javascripts/components/widget-menu.js

```diff
@@ -0,0 +1,37 @@
+ManageIQ.angular.app.component('widgetMenu', {
+  bindings: {
+    id: '<',
+  },
+  controllerAs: 'vm',
+  controller: ['$http', 'miqService', function($http, miqService) {
+    var vm = this;
+    vm.widgetMenuModel = {shortcuts: []};
+    vm.shortcutsMissing = function() {
+      return vm.widgetMenuModel.shortcuts.length === 0;
+    };
+    this.$onInit = function() {
+      $http.get('/dashboard/widget_menu_data/' + vm.id)
+        .then(function(response) { vm.widgetMenuModel = response.data; })
+        .catch(miqService.handleFailure);
+      vm.div_id = 'dd_w' + vm.id + '_box';
+    };
+  }],
+  template: [
+    '<div class="mc" id={{vm.div_id}} ng-style="{ \'display\' : (vm.widgetMenuModel.minimized === true) ? \'none\' : \'\' }">',
```

Comment:
quotes around the id please ;)

And can you replace that ng-style with `ng-class="{ hidden: vm.widgetMenuModel.minimized }"`?